spec/schemas/security._common.yaml

```diff
@@ -170,16 +178,29 @@ components:
           type: string
           description: Size of backend roles in bytes.
 
-    BadRequest:
+    # BadRequest:
+    #   type: object
+    #   properties:
+    #     status:
+    #       type: string
+    #       enum:
+    #         - 400
+    #     message:
+    #       type: string
+    #       description: Message returned as part of BAD_REQUEST response.
```

Comment:
Let's get rid of the comments. This one likely needs to be reinstated.

Reply:
I have kept them so it might easier to uncomment once status other than 200 are supported.